src/main/java/teammates/ui/template/FeedbackSubmissionEditQuestion.java

```diff
@@ -35,6 +38,9 @@ public FeedbackSubmissionEditQuestion(FeedbackQuestionAttributes questionAttribu
         isRecipientNameHidden = questionAttributes.isRecipientNameHidden();
         isGiverTeam = questionAttributes.giverType.equals(FeedbackParticipantType.TEAMS);
         isRecipientTeam = questionAttributes.recipientType.isTeam();
+        isRecipientOwnTeamMembers = questionAttributes.recipientType.isOwnTeamMembers();
+        isRecipientOtherTeams = questionAttributes.recipientType.isOtherTeams();
```

Comment:
Yeah, I need to use this in jsp, to check which message I need to display (If there are no responses boxes). There are 3 possibilities in which student get no response box for the question. 
1. Question for Team members (not including self), and user don't have any team members.
2. Question for Other teams (excluding self team), and course doesn't have any other Team in course.
3. Question for Other student (excluding self), and course doesn't have any other student in course.

So, I have defined 1 boolean for each case here. I am using these to check, which message to be displayed.
Is there some other way to check these, without defining booleans for each ?